src/main/java/org/folio/spring/FolioExecutionContext.java

```diff
@@ -2,14 +2,18 @@
 
 import java.util.Collection;
 import java.util.Map;
+import java.util.UUID;
 
 public interface FolioExecutionContext {
+
   String getTenantId();
 
   String getOkapiUrl();
 
   String getToken();
 
+  UUID getUserId();
```

Comment:
It seems we have to bump minor version, because it seems to me a breaking change, or maybe do something like:
```suggestion
  default UUID getUserId() {
    return null;
  }
```

What do you think?